lib/widget/image.dart

```diff
@@ -154,6 +163,16 @@ class ImageState extends WidgetState<EnsembleImage> {
     return "${widget.controller.source}${str}timeStamp=$lastModifiedDateTime";
   }
 
+  Widget buildMemoryImage(Uint8List source) {
```

Comment:
does this work if it's png or svg?

Reply:
It's working fine for PNG and JPEG formats but it not working for SVG